odata-json-format/19 Batch Requests and Responses.md

```diff
@@ -262,6 +262,48 @@ Content-Length: ###
 ```
 :::
 
+## ##subsec Referencing Response Body Values
+
+::: example
+Example ##ex: a batch request that contains the following operations in
+the order listed:
+the order listed:
+- Get an employee (with `Content-ID = 1`)
+- Get all employees residing in the same building
```

Comment:
Get? The second request is a PATCH request.

Reply:
Thanks! Forgot to completely "patch" the copied example.